### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by bih4st** [rodit/SnapMod#255] [Q&A/Support]
A lot of problems are happening in the new update
- Failed to send snap to friend and story
- Tap to your story not working
- somtimes your friend chat not showing
- sometimes chat menu not showing
- and other..

**Comment by 0xFFFFail** [comment]
Can confirm about sending video snaps of any length, won't send.
On occasion too tapping into a friends chat brings up a blank chat screen.

**Reply by RevealedSoulEven** [reply]
yes. True

**Comment by RevealedSoulEven** [comment]
Yes. These all bugs are happening

**Reply by RSR1337** [reply]
> Yes. These all bugs are happening

True, there are many bugs , back to 1.8.2...

**Comment by jib0o** [comment]
There is a problem when sending a video clip of more than 10 seconds from the gallery to Snap, it is not possible to do so

Thank you

**Reply by RevealedSoulEven** [reply]
I even can't send normal snap videos even shorter than 10s

**Comment by ozilwg** [comment]
> A lot of problems are happening in the new update
> 
> * Failed to send snap to friend and story
> * Tap to your story not working
> * somtimes your friend chat not showing
> * sometimes chat menu not showing
> * and other..

I have the same bugs too.  In addition, a random message is left in the chat, which cannot be canceled or deleted

**Reply by RevealedSoulEven** [reply]
Yes
